packages/test/test-end-to-end-tests/src/test/mapEndToEndTests.spec.ts

```diff
@@ -341,10 +341,14 @@ describeNoCompat("SharedMap orderSequentially", (getTestObjectProvider) => {
     beforeEach(() => {
         provider = getTestObjectProvider();
     });
+    afterEach(async () => reset());
```

Comment:
can probably just do `afterEach(reset)`?

Reply:
And likewise below do `const reset = provider.reset;`

Reply:
Also... this may happen automatically somehow...?  Not sure actually.

Reply:
I run into some reference errors when I try this, namely provider being used before its assigned